ethereum/test/test_beefy_light_client_real_fixture_data.js

```diff
@@ -59,12 +60,33 @@ describe("Beefy Light Client", function () {
     await mine(45);
 
     const bitfield = await this.beefyLightClient.createRandomBitfield(lastId);
-    expect(printBitfield(bitfield)).to.eq('11')
+    const bitfieldString = printBitfield(bitfield);
+    const bitfieldIndex = [Number(bitfield)] - 1
```

Comment:
What's the idea of subtracting a number from an array here?

Reply:
I just checked and it works due to some JavaScript conversion magic, but is there an actual reason for this?

Reply:
err i actually have no idea what i was doing here, probably a typo/mistake that didn't actually result in a failure

Reply:
fixed here: https://github.com/Snowfork/snowbridge/pull/454/files#diff-55c3a46404a460a09847d296e249c0e87d0a9cd4741eccda7838c4b0adaae78fR64